tests/unit/states/test_libvirt.py

```diff
@@ -23,13 +25,30 @@
 import salt.utils.files
 
 
+class LibvirtMock(MagicMock):  # pylint: disable=too-many-ancestors
+    '''
+    libvirt library mockup
+    '''
+
+    class libvirtError(Exception):  # pylint: disable=invalid-name
+        '''
+        libvirt error mockup
+        '''
+        pass
```

Comment:
No need this after you have a docstring. :wink:

Reply:
Oh ok, I'll remove it then